examples/variables-1.md

```diff
@@ -41,7 +41,7 @@ GROUP BY
   title,
   country
 HAVING
-  ct > 200
+  COUNT(*) > 200
 ORDER BY
-  sum_gross_cost
+  SUM(salary + payroll_tax + benefits_cost)
```

Comment:
These changes are fine — they still work. I think the original ones are a bit easier to read, because they use the variable names.

(but this is 5% of the weight against the huge improvements, and is easy to leave to another day)

Reply:
Awesome!